doc/tutorials/sandbox_config_ui.ipynb

```diff
@@ -0,0 +1,91 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.            'bfs service':  widgets.Text(description='BucketFS Service', value=sb_config.get('BUCKETFS_SERVICE', 'bfsdefault'), style=style),
</pre><p>I think, with BucketFS Service Name, it  is more clear what we mean. Otherwise, they may enter the hostname again</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>